Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:22.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -qq -y \
+    git \
+    python3 \
+    python3-pip && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* 
+#&& pip install mse-cli
+
+RUN git clone https://github.com/Cosmian/mse-cli
```

Comment:
why not `COPY . /mse-cli` ?

Reply:
I will install it later without cloning the repo but by using the pip package